setupUrlParams.test.js

```diff
@@ -15,14 +15,14 @@ describe('setupUrlParams', () => {
       const mockGetPath = () => '/fake-id/list'
       const useUrlParamsWithMock = setupUrlParams(mockGetPath)
       const params = useUrlParamsWithMock('/:account/page')
-      expect(params).toBeFalsy()
+      expect(params.matches).toBeFalsy()
```

Comment:
If the test after this is matching on `{matches: true}` then a good pattern to follow for this and the other false assertion would be: `{ matches: false }`. They should all be toEqual to be consistent